config/initializers/download_processor.rb

```diff
@@ -0,0 +1,4 @@
+module Gemcutter
+  # Feature flag for counting downloads through stats-update or FastlyLogProcessor
+  ENABLE_FASTLY_LOG_PROCESSOR = Rails.env.test? || Rails.env.development?
```

Comment:
why do we need this constant here? can we move this to the Download model or the log processor itself?